### PR TITLE
Added Swedish language support.

### PR DESCRIPTION
**Pull request by ejulen**
I thought this was all fairly straightforward except for the meridiem stuff. As far as I know, we Swedes don't really use that kind of notation very often, and the only times I _have_ seen it it's always been in the form of "f.m." (AM) and "e.m." (PM). Never uppercase or written without the periods. So that's what I did, no matter if you use the `A`, `a` or `aa` formatting.

### CODE REVIEW

**Comment by ejulen** [comment]
@LeshaKoss will do.

**Comment by LeshaKoss** [comment]
@ejulen thank you 🙏

File: src/locale/sv/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var buildDistanceInWordsLocale = require('./build_distance_in_words_locale/index.js')
+var buildFormatLocale = require('./build_format_locale/index.js')
+
+// English
```
Comment (kossnocorp) [inline src/locale/sv/index.js]: I think it's redundant 😊

Comment (LeshaKoss) [inline src/locale/sv/index.js]: 😀 @ejulen Please change it to `// Swedish`

**Comment by kossnocorp** [comment]
@LeshaKoss :shipit:

**Comment by LeshaKoss** [comment]
@ertrzyiks thanks you for your contribution! 🇸🇪 Swedish locale was released with [1.17.0](https://github.com/date-fns/date-fns/releases/tag/v1.17.0) (diff: https://github.com/date-fns/date-fns/compare/v1.16.0...v1.17.0)

**Comment by LeshaKoss** [comment]
@ejulen thanks you for your contribution! 🇸🇪 Swedish locale was released with [1.17.0](https://github.com/date-fns/date-fns/releases/tag/v1.17.0) (diff: https://github.com/date-fns/date-fns/compare/v1.16.0...v1.17.0)

(sorry for copypaste error in the previous message 😅)

**Comment by micke232** [comment]
@ejulen hey! I was wondering why `'dd MMM'` gives me a period after, let's say ` jan. ` is this some standard? I'm having a hard time to find this particular standard. Could you help me out?

**Comment by ejulen** [comment]
Apologies, I wrote some of this several years ago so my memory is pretty 
hazy, I don't recall finding a standard for this some of these formats but 
rather applied formatting I had seen in the wild. If you have some other 
preference, especially one linked to some standard, I'm sure no one would 
object to a PR.

**Comment by fturmel** [comment]
@micke232 Unicode CLDR is usually the reference for locales in this project.

v32 (linked in the docs): https://www.unicode.org/cldr/cldr-aux/charts/32/summary/sv.html
v42 (latest): https://unicode-org.github.io/cldr-staging/charts/latest/summary/sv.html

Screenshot 2023-01-24 at 10 58 07 AM

**Comment by micke232** [comment]
Thanks for the replies! @ejulen @fturmel

**Pull request merged**